### PR TITLE
MW-172: Fixed errors appearing in the console while viewing requisition

### PR DESCRIPTION
**Pull request by weronika-ciecierska**

### CODE REVIEW

File: src/requisition-full-supply/full-supply.controller.js [hunk 1]
```diff
@@ -171,6 +171,20 @@
...
+         * @ngdoc method
+         * @methodOf requisition-full-supply.controller:FullSupplyController
+         * @name canEditRequisition
+         *
+         * @description
+         * Determines whether the requisition is editable.
+         *
+         * @return {Boolean} true if requisition is editable
+         */
+        function canEditRequisition() {
```
Comment (ngraczewski) [inline src/requisition-full-supply/full-supply.controller.js]: Is there any way we could avoid this code duplication?

Comment (weronika-ciecierska) [inline src/requisition-full-supply/full-supply.controller.js]: I don't know if there is any simple way to do this - do you have any suggestions?

Comment (nickdotreid) [inline src/requisition-full-supply/full-supply.controller.js]: I think this is unduplicated enough... for now

the problem is the markup that needs these functions exposed — which could be replaced with direct calls to vm.requisition.$isEditable.

Main functionality is abstracted back to a single point that is easy to extend

@ngraczewski — I'd megre this, but I'd like you to agree

Comment (ngraczewski) [inline src/requisition-full-supply/full-supply.controller.js]: Merged :)

**Pull request merged**